src/instructlab/model/accelerated_train.py

```diff
@@ -77,17 +82,29 @@ def accelerated_train(
             pprint.pformat(train_args),
         )
 
-        if not (phased_phase1_data and phased_phase2_data):
-            # pylint: disable=broad-exception-raised
-            raise Exception(
-                "End-to-end training minimally requires: `--phased-phase1-data`, and `--phased-phase2-data`. One or more wasn't correctly specified."
-            )
-
-        # if they both exist, must be Path objects
-        if not (phased_phase1_data.exists() and phased_phase2_data.exists()):
-            raise FileNotFoundError(
-                "Data for both phase1 and phase2 must be specified for phased training."
-            )
+        if strategy == SupportedTrainingStrategies.LAB_MULTIPHASE.value:
```

Comment:
One minor question is whether we'd like to take this opportunity to refactor our use of enums since we're starting to use them more. Stringifying enums in python this way is one way to accomplish this, another would be:

`if SupportedTrainingStrategies(strategy) == SupportedTrainingStrategies.LAB_MULTIPHASE`

Which is obviously less concise but seems like that the enum docs prefer.... I don't have a strong opinion either way, what do you think?

Reply:
I was just trying to be consistent with existing code.  I prefer not to use .value as well.  Although I think would prefer to change in a follow-on PR if that ok?